testdata/broadcast/Counter.s.sol/1/run-1729591886.json

```diff
@@ -0,0 +1 @@
000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","type":"0x2","transactionHash":"0xf36fa2251627aead1719f8965161fb6ef4bffa5ac804753609f83f3a9f8a5f55","transactionIndex":"0x0","blockHash":"0x413481362125fe1ab076e83359c5f447c67dd9f914cef28e37037d8730ae11c9","blockNumber":"0x1","gasUsed":"0x1a6d5","effectiveGasPrice":"0x3b9aca01","blobGasPrice":"0x1","from":"0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266","to":null,"contractAddress":"0x5fbdb2315678afecb367f032d93f642f64180aa3"},{"status":"0x1","cumulativeGasUsed":"0xa998","logs":[],"logsBloom":"0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","type":"0x2","transactionHash":"0x1046225989e019746b245e8b746933795371efe39c946f3b18bc43d70e260d9f","transactionIndex":"0x0","blockHash":"0x691195e092ea564cb9903a0a749bf990ec5a68b3deb162a6fc5e31a292c8a93c","blockNumber":"0x2","gasUsed":"0xa998","effectiveGasPrice":"0x3435345b","blobGasPrice":"0x1","from":"0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266","to":"0x5fbdb2315678afecb367f032d93f642f64180aa3","contractAddress":null},{"status":"0x1","cumulativeGasUsed":"0x110e8","logs":[],"logsBloom":"0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","type":"0x2","transactionHash":"0x89bb744ef5efc42501cb66da49548ab72526d936fb5a5e3f181ac34def638d16","transactionIndex":"0x1","blockHash":"0x691195e092ea564cb9903a0a749bf990ec5a68b3deb162a6fc5e31a292c8a93c","blockNumber":"0x2","gasUsed":"0x6750","effectiveGasPrice":"0x3435345b","blobGasPrice":"0x1","from":"0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266","to":"0x5fbdb2315678afecb367f032d93f642f64180aa3","contractAddress":null}],"libraries":[],"pending":[],"returns":{},"timestamp":1729591886,"chain":1,"commit":"50dd68a"}
```

Comment:
I've meant adding a CLI test which would both produce and read artifacts, but this should be fine too

Reply:
Added cli test